.ci/env/editorconfig-checker.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#===============================================================================
+# Copyright contributors to the oneDAL project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+VERSION=v3.0.3
```

Comment:
is there a way to set it as automatically updatable version?  through automatic tool updates?

Reply:
have updated `renovate.json` config for this purpose: https://github.com/oneapi-src/oneDAL/pull/2849/commits/fe85bbea12c062d53bc270240a677726d440b4c8